metsrw/plugins/dcrw/dc.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import absolute_import
+
+from collections import OrderedDict
+import logging
+from lxml import etree
+
+from .exceptions import ParseError
+from .utils import lxmlns, NAMESPACES, DUBLINCORE_SCHEMA_LOCATIONS
+
+LOGGER = logging.getLogger(__name__)
+
+
+class DublinCoreXmlData(object):
+    """
+    An object representing a METS xmlData element containing a Dublin Core element.
+
+    :raises ParseError: If the root element tag is not xmlData.
+    """
+    DC_ELEMENTS = ['title', 'creator', 'subject', 'description', 'publisher', 'contributor', 'date', 'format', 'identifier', 'source', 'relation', 'language', 'coverage', 'rights']
+
+    def __init__(self, title=None, creator=None, subject=None, description=None, publisher=None, contributor=None, date=None, format=None, identifier=None, source=None, relation=None, language=None, coverage=None, rights=None):
```

Comment:
Comment by @Hwesta: This is a great use case for kwargs instead of accessing locals() directly. We could replace the list of parameters with **kwargs, which would allow us to accept an arbitrary number of inputs. kwargs is a dictionary. Then the loop could be setattr(self, element, kwargs[element])